orte/mca/rml/ofi/rml_ofi_component.c

```diff
@@ -258,13 +325,17 @@ void print_provider_list_info (struct fi_info *fi )
  * This returns all the supported transports in the system that support endpoint type RDM (reliable datagram)
  * The providers returned is a list of type opal_valut_t holding opal_list_t
  */
-static int orte_rml_ofi_query_transports(opal_value_t **providers)
+/*[Anandhi] old defn static int orte_rml_ofi_query_transports(opal_value_t **providers) */
+static orte_rml_pathway_t* query_transports(void)
 {
     opal_list_t *ofi_prov = NULL;
     opal_value_t *providers_list, *prev_provider=NULL, *next_provider=NULL;
     struct fi_info *cur_fi = NULL;
-    int ret = 0, prov_num = 0;
+    int ret = 0, ofi_prov_num = 0;
 
+ opal_output_verbose(10,orte_rml_base_framework.framework_output,
+                        "\n%s:%d OFI Query Interface not implemented",__FILE__,__LINE__);
```

Comment:
Verbose output should not include a `\n` to put it on the next line (i.e., separated from its prefix).